website/docs/guides/orchestration/set-up-ci/2-quick-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Quickstart: Enable CI in 15 minutes"
```

Comment:
we can't call non-Quickstarts quickstarts that's a specific format

Reply:
~Quickstart~ Baseline